src/components/pages/landing/FractalVerification.js

```diff
@@ -12,11 +12,11 @@ import { supabase } from '../../../utils/supabase';
 import { log_event } from '../../../utils/utilityFunctions';
 import { getConfig } from '../../../utils/config';
 
-export const Gooddollar = ({ setShowGooddollarVerification }) => {
+export const FractalVerification = ({ setShowGooddollarVerification }) => {
   const [editableFields, setEditableFields] = useState({
     code: '',
     claimer: wallet.accountId,
-    redirect_uri: '',
+    redirect_uri: 'https://i-am-human-dev.netlify.app/',
```

Comment:
can we assign it to constant, and use it according to testnet and mainnet env (since we have different links for both)

Reply:
> ot able to verify after I get the email, I click on t

Why do you check all the functionality (flow)? I noted in the MR description exactly what task I did here